compiler/one-cmds/tests/onecc_013.test

```diff
@@ -30,7 +30,8 @@ trap trap_err_onexit ERR
 configfile="onecc_013.cfg"
 outputfile="inception_v3.circle"
 
-rm -rf ${outputfile}
+# do not remove output file
+# rm -rf ${outputfile}
```

Comment:
Can I ask why outputfile is not removed in `onecc_013.test` and `onecc_014.test`?

If outputfile exists before test is started, the test will be always passed.

Reply:
this test conflicts with the problem deleing reused `inception_v3.circle` file.
I'll try to rename the output file